temoa/extensions/economies_of_scale/core/model.py

```
@@ -0,0 +1,208 @@
+    Any,
+    Binary,
+    BuildAction,
+    Constraint,
+    Integers,
+    NonNegativeReals,
+    Param,
+    Set,
+    Var,
+)
+
+import temoa.extensions.economies_of_scale.components.cost_fixed_eos as cost_fixed_eos
```

Ruff (0.15.21)

[warning] Use `from temoa.extensions.economies_of_scale.components import cost_fixed_eos` in lieu of alias

Replace with `from temoa.extensions.economies_of_scale.components import cost_fixed_eos`

(PLR0402)